src/main/scala/itc/ItcObservationDetails.scala

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) 2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package basic.itc
+
+import io.circe.{ Encoder, Json }
+import io.circe.generic.semiauto._
+
+final case class ItcObservationDetails(
+  calculationMethod: ItcObservationDetails.CalculationMethod,
+  analysisMethod:    ItcObservationDetails.AnalysisMethod
+)
+
+object ItcObservationDetails {
+
+  sealed trait CalculationMethod
+    extends Product with Serializable
+
+  object CalculationMethod {
+
+    trait SignalToNoise extends CalculationMethod
+    object SignalToNoise {
+
+      final case class Imaging(
+        exposures:      Int,
+        coadds:         Option[Int],
+        exposureTime:   Double,
```

Comment:
Is there a reason we need raw `Double`s for these things instead of, say, a Java `Duration` or a Scala `FiniteDuration`?

Reply:
ITC speaks in terms of `Double` but I can switch to use `FiniteDuration` here. It's just a question of how much we want to diverge from the OCS2 model down here in the glue code.